src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataParser;
+import core.basesyntax.service.ReportCreator;
+import core.basesyntax.service.ReportReader;
+import core.basesyntax.service.ReportWriter;
+import core.basesyntax.service.impl.DataParserImpl;
+import core.basesyntax.service.impl.ReportCreatorImpl;
+import core.basesyntax.service.impl.ReportReaderImpl;
+import core.basesyntax.service.impl.ReportWriterImpl;
+import core.basesyntax.strategy.BalanceStrategy;
+import core.basesyntax.strategy.DecreaseStrategy;
+import core.basesyntax.strategy.FruitStrategy;
+import core.basesyntax.strategy.IncreaseStrategy;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_PATHNAME = "src/main/resources/data.csv";
+    private static final String REPORT_PATHNAME = "src/main/resources/report.csv";
```

Comment:
I don't know is my suggestion valid. 
I would move it to ReportWriter and ReportReader.

Reply:
I wrote them in main so I could change them there if needed)